src/utils/internal.ts

```diff
@@ -1318,7 +1318,7 @@ function handleSpecVersionResponse(
     return false;
   }
 
-  if (!satisfies(specVersionAsSemver, SUPPORTED_SPEC_VERSION_RANGE)) {
+  if (gtr(specVersionAsSemver, SUPPORTED_SPEC_VERSION_RANGE)) {
```

Comment:
By changing it to `gtr` this will only log when specVersion from chain is greater than the specified range